mongo/mongo.go

```diff
@@ -57,6 +57,7 @@ func BuildSession(ctx context.Context, rawURL string) (*mgo.Session, error) {
 	if err != nil {
 		return nil, err
 	}
+	info.Timeout = 10 * time.Second
```

Comment:
Do we want to add a way to specify an argument in the connection string to customize this? Such as `mongodb://...?timeout=5s`

Reply:
Done